### DISCUSSION TITLE
Got error when using jsonRpcProvider for BSC testnet

### DISCUSSION THREAD

**Discussion by huaigu** [wevm/wagmi#1304] [Q&A]
1. config jsonRpcProvider  in app.ts

`const bscTestNetChain = {
  id: 97,
  name: "Binance Smart Chain Testnet",
  network: "BSC",
  nativeCurrency: {
    decimals: 18,
    name: "tBNB",
    symbol: "tBNB",
  },
  rpcUrls: {
    default: "https://rpc.ankr.com/bsc_testnet_chapel",
  },
  blockExplorers: {
    default: { name: "SnowTrace", url: "https://testnet.bscscan.com" },
  },
  testnet: true,
};`

then I got this

`Refused to connect to 'https://rpc.ankr.com/bsc_testnet_chapel' because it violates the following Content Security Policy directive: "connect-src 'self' wss://*.alchemyapi.io wss://*.alchemy.com https://*.alchemy.com https://*.infura.io wss://*.walletconnect.org https://*.vercel-insights.com".`

**Comment by huaigu** [comment]
my problem, in next config, please delete it.

**Reply by achal-singh** [reply]
How did you resolve the error?
